src/base-widget.ts

```diff
@@ -25,7 +33,29 @@ export class Widget {
 export type Connector = (
   renderFn: (state: object, isFirstRendering: boolean) => void,
   unmountFn: () => void
-) => (widgetOptions?: object) => Widget;
+) => (widgetParams?: object) => Widget;
+
+export const connectNoop: Connector = function(
```

Comment:
I would have left it in user land, but since you did a good job of typing it etc, and it's harder than I remember, having it in the library also kind of makes sense.

Reply:
I moved it in user land 😄